apis/python/src/tiledbsoma/_sparse_nd_array.py

```diff
@@ -463,6 +473,21 @@ class SparseNDArrayRead(_SparseNDArrayReadBase):
         Maturing.
     """
 
+    def dask_array(
+        self,
+        **config: Unpack[DaskConfig],
+    ) -> "da.Array":
+        """Load a TileDB-SOMA X layer as a Dask array."""
```

Comment:
lifecycle tags missing - this is definitely an experimental API

Reply:
done